client/dashboard/app/style.scss

```diff
@@ -5,6 +5,8 @@
 // I'm tempted to even just replace all this with a display:none on the logo
 @import './loading-screen';
 
+@import './view-transitions';
```

Comment:
I like that it feels not intrusive like that. It's its own CSS file that we can remove and iterate on as needed. I don't like that we have to use classnames for things that are rendered in different places to achieve it though.

I'm not sure yet what's the best approach.